docs/tutorial/deploy.md

```diff
@@ -3,147 +3,183 @@
 
 This is a part of the [Charmed Apache Kafka Tutorial](index.md).
 
-## Deploy Charmed Apache Kafka (and Charmed ZooKeeper)
+## Deploy Charmed Apache Kafka
 
-To deploy Charmed Apache Kafka, all you need to do is run the following commands, which will automatically fetch [Apache Kafka](https://charmhub.io/kafka?channel=3/stable) and [Apache ZooKeeper](https://charmhub.io/zookeeper?channel=3/stable) charms from [Charmhub](https://charmhub.io/) and deploy them to your model. For example, to deploy a cluster of five Apache ZooKeeper units and three Apache Kafka units, you can simply run:
+To deploy Charmed Apache Kafka, all you need to do is run the following commands, which will automatically fetch [Apache Kafka](https://charmhub.io/kafka?channel=4/edge) from [Charmhub](https://charmhub.io/) and deploy it to your model.
+
+For example, to deploy a cluster of three Apache Kafka brokers, you can simply run:
+
+```shell
+$ juju deploy kafka -n 3 --channel 4/edge --roles=broker
```

Comment:
Chore: We should avoid using the `$` prompt (or any other prompt) in the console/shell/bash code blocks.